static/app/components/organizations/timeRangeSelector/dateRange/index.tsx

```diff
@@ -195,15 +198,19 @@ class BaseDateRange extends Component<Props, State> {
         />
         {showTimePicker && (
           <TimeAndUtcPicker>
-            <TimePicker
+            <StyledTimePicker
               start={startTime}
               end={endTime}
               onChangeStart={this.handleChangeStart}
               onChangeEnd={this.handleChangeEnd}
             />
             <UtcPicker>
-              {t('Use UTC')}
-              <Checkbox onChange={onChangeUtc} checked={utc || false} />
+              <Checkbox
+                onChange={onChangeUtc}
+                checked={utc || false}
+                id={this.utcInputId}
+              />
+              <UtcPickerLabel htmlFor={this.utcInputId}>{t('UTC')}</UtcPickerLabel>
```

Comment:
So we can click on the text label to toggle the checkbox.
<img width="67" alt="image" src="https://user-images.githubusercontent.com/44172267/233491585-87fabc94-8976-4c59-910c-6730c94a8ac9.png">